care/facility/management/commands/load_redis_index.py

```diff
@@ -17,6 +17,17 @@ class Command(BaseCommand):
     help = "Loads static data to redis"
 
     def handle(self, *args, **options):
+        try:
+            deleted_count = self.delete_pattern("care_static_data*")
```

Comment:
```suggestion
            deleted_count = cache.delete_pattern("care_static_data*")
```

Reply:
Thank you for the feedback. I've changed it to...
```deleted_count = cache.delete_pattern("care_static_data*", itersize=25_000)```